components/backup-stream/src/observer.rs

```diff
@@ -149,7 +156,7 @@ impl<E: KvEngine> CmdObserver<E> for BackupStreamObserver {
 
 impl RoleObserver for BackupStreamObserver {
     fn on_role_change(&self, ctx: &mut ObserverContext<'_>, r: &RoleChange) {
-        if r.state != StateRole::Leader {
+        if r.state != StateRole::Leader && !self.is_hibernating() {
```

Comment:
We need to test the full process containing the log-backup and PITR, check data consistency finally, after this PR is merged into master.